### ISSUE TITLE
The circleci test is broken.

### ISSUE THREAD

**Issue by charris** [numpy/numpy#10595]
I suspect this is related to the recent release of sphinx 1.7.0 on 2/12/2018. We may need to update the submodule hash for sphinxext, or maybe even fix numpydoc.

**Comment by pv** [comment]
The doc/sphinxext submodule needs to be bumped to the v0.7.0 tag in 
numpydoc.

**Comment by charris** [comment]
Will do, thanks for the heads up Pauli.

**Comment by charris** [comment]
New error with Python 3.6
```
Exception occurred:
  File "/home/circleci/repo/doc/sphinxext/numpydoc/docscrape.py", line 343, in _parse
    raise ValueError(msg)
ValueError: The section Notes appears twice in the docstring of None in None.
```
Don't know where that came from yet.

**Comment by pv** [comment]
numpy.ma.inner docstring

**Comment by charris** [comment]
Hmm, looks like `doc_note` in `numpy/ma/core.py` is buggy.

**Comment by charris** [comment]
@pv Might be useful if numpydoc could merge duplicate Notes sections.

**Comment by charris** [comment]
Fixed by #10596.
